sympy/matrices/sparse.py

```diff
@@ -403,6 +408,10 @@ def _eval_transpose(self):
     def _eval_values(self):
         return [v for k,v in self._smat.items() if not v.is_zero]
 
+    @classmethod
+    def _eval_zeros(cls, rows, cols):
```

Comment:
I didn't find it called anywhere. Can it be called only after all subclasses have it defined?